test/packer_basic_example_test.go

```diff
@@ -28,5 +28,13 @@ func TestPackerBasicExample(t *testing.T) {
 	amiID := packer.BuildAmi(t, packerOptions)
 
 	// Clean up the AMI after we're done
-	defer aws.DeleteAmi(t, awsRegion, amiID)
+	defer deleteAmi(t, awsRegion, amiID)
+}
+
+func deleteAmi(t *testing.T, awsRegion string, amiID string) {
```

Comment:
Perhaps we should add a `DeleteAmiAndAllSnapshots` method to the Terratest API that does exactly this?